substrate/frame/staking/src/migrations.rs

```diff
@@ -60,6 +60,36 @@ impl Default for ObsoleteReleases {
 #[storage_alias]
 type StorageVersion<T: Config> = StorageValue<Pallet<T>, ObsoleteReleases, ValueQuery>;
 
+pub mod v16 {
+	use super::*;
+
+	pub struct VersionUncheckedMigrateV15ToV16<T>(core::marker::PhantomData<T>);
+	impl<T: Config> UncheckedOnRuntimeUpgrade for VersionUncheckedMigrateV15ToV16<T> {
+		fn on_runtime_upgrade() -> Weight {
+			// Set the default unbonding queue params.
+			UnbondingQueueParams::<T>::set(Default::default());
```

Comment:
There is no need to do this. If the value is none, default will be used.
I don't think there are any storage breaking changes so no need migration / version bump/

Reply:
Excellent, that simplifies things a lot, I will remove this then. Originally the RFC were contemplated block numbers in storage but we've have since updated things to maintain all storage related data as `EraIndex`.